src/index.html

```diff
@@ -4,13 +4,59 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar">
+      <form
+        action="https://server.api"
+        method="get"
+        data-qa="big"
+        class="big-form"
+      >
+        <label class="label">
+          <input
+            id="big_input"
+            data-qa="keypress"
+            type="text"
+            name="search"
+            placeholder="Try &quot;Los Angeles&quot;"
+            class="input"
+            tabindex="1"
+          >
+          <img
+            src="./images/Search.svg"
+            alt="search_icon"
+            class="icon"
+          >
+        </label>
+      </form>
+
+      <form
+        action="https://server.api"
+        method="get"
+        data-qa="small"
+        class="small-form">
+        <label class="label"
+      >
```

Comment:
```suggestion
        <label class="label">
```

Fix this line please

Reply:
Thanks, It is done